src/pages/household/input/TaxYear.jsx

```diff
@@ -71,7 +71,7 @@ export default function TaxYear(props) {
   }
 
   return (
-    <CenteredMiddleColumn title="Which tax year would you like to calculate?">
+    <CenteredMiddleColumn title="Which year would you like to calculate?">
```

Comment:
read my mind lol

Reply:
Nikhil had the exact same feedback - great minds think alike